locallibrary/catalog/models.py

```diff
@@ -1,3 +1,85 @@
 from django.db import models
+from django.urls import reverse
+import uuid
+
 
 # Create your models here.
+class Genre(models.Model):
+    name = models.CharField(
+        max_length=200, help_text="Enter the book genre (e.g: SF, Romantic,...)"
```

Comment:
` help_text="Enter the book genre (e.g: SF, Romantic,...)"`
-> E tìm hiểu kỹ thuật I18n - đa ngôn ngữ trong Django và apply cho project nha
Với các static text trong file Model, View, Template e cần phải I18n hết nhé
VD trong file model là các message `help_text` nha